flink-end-to-end-tests/test-scripts/common_ha.sh

```diff
@@ -23,10 +23,9 @@ source "${END_TO_END_DIR}"/test-scripts/common.sh
 # flag indicating if we have already cleared up things after a test
 CLEARED=0
 
-JM_WATCHDOG_PID=0
```

Comment:
Just a curiosity, what's the reason of this change?

Reply:
The `JM_WATCHDOG_PID` is used in the `stop_watchdogs` function, which is called from different bash-processes.
Because of that, if the variable is set in one process, it's not available in the other one.

That's why I decided to store the PID in a file.